Unable to scrape properly with 1.9.3-rc3

```
mwoffliner --adminEmail="contact@kiwix.org" --format="nopic" --format="novid" --mwUrl="https://en.wikipedia.org/" --customZimFavicon="https://upload.wikimedia.org/wikipedia/commons/8/84/Biofilm.png" --customZimTitle="Movies by Wikipedia" --customZimDescription="A selection of Wikipedia articles on movies" --articleList="http://download.kiwix.org/wp1/enwiki/customs/movies" --useCache --redis="redis://redis" --outputDirectory="/output"
internal/modules/cjs/loader.js:638
    throw err;
    ^

Error: Cannot find module 'src/stores'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:636:15)
    at Function.Module._load (internal/modules/cjs/loader.js:562:25)
    at Module.require (internal/modules/cjs/loader.js:690:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.<anonymous> (/usr/lib/node_modules/mwoffliner/lib/util/rewriteUrls.js:24:18)
    at Module._compile (internal/modules/cjs/loader.js:776:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:787:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Module.require (internal/modules/cjs/loader.js:690:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.<anonymous> (/usr/lib/node_modules/mwoffliner/lib/util/saveArticles.js:33:23)
    at Module._compile (internal/modules/cjs/loader.js:776:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:787:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
```

--- a/src/util/rewriteUrls.ts
+++ b/src/util/rewriteUrls.ts
@@ -1,11 +1,11 @@
 import * as urlParser from 'url';
 import { isMirrored } from './saveArticles';
 import { migrateChildren, getMediaBase, getFullUrl } from './misc';
-import { redirectsXId } from 'src/stores';
-import { Dump } from 'src/Dump';
-import MediaWiki from 'src/MediaWiki';
+import { redirectsXId } from '../stores';
+import { Dump } from '../Dump';
+import MediaWiki from '../MediaWiki';
 import DU from '../DOMUtils';
-import logger from 'src/Logger';
+import logger from '../Logger';
 
 export async function removeLinksToUnmirroredArticles(mw: MediaWiki, dump: Dump, linkNode: DominoElement, href: string) {
     const title = mw.extractPageTitleFromHref(href);

--- a/test/e2e/bm.e2e.test.ts
+++ b/test/e2e/bm.e2e.test.ts
@@ -1,5 +1,5 @@
 import test from 'blue-tape';
-import { execute } from 'src/mwoffliner.lib';
+import { execute } from '../../src/mwoffliner.lib';
 
 import rimraf from 'rimraf';
 // import { ZimReader } from '@openzim/libzim';

--- a/test/e2e/en10.e2e.test.ts
+++ b/test/e2e/en10.e2e.test.ts
@@ -1,5 +1,5 @@
 import test from 'blue-tape';
-import { execute } from 'src/mwoffliner.lib';
+import { execute } from '../../src/mwoffliner.lib';
 import { leftPad } from '../util';
 import rimraf from 'rimraf';
 // import { ZimReader } from '@openzim/libzim';

--- a/test/unit/sanity.test.ts
+++ b/test/unit/sanity.test.ts
@@ -2,7 +2,7 @@
 ///<reference path="../../src/types.d.ts" />
 
 import test from 'blue-tape';
-import { contains } from 'src/util';
+import { contains } from '../../src/util';
 
 // Super Simple Sanity tests
 test(async (t) => {

--- a/test/unit/saveArticles.test.ts
+++ b/test/unit/saveArticles.test.ts
@@ -9,9 +9,9 @@ import domino from 'domino';
 
 import { setupScrapeClasses } from 'test/util';
 import { populateArticleDetail, articleDetailXId, populateRedirects, redirectsXId, populateFilesToDownload, filesToDownloadXPath } from 'src/stores';
-import Redis from 'src/redis';
-import { config } from 'src/config';
-import { saveArticles } from 'src/util/saveArticles';
+import Redis from '../../src/redis';
+import { config } from '../../src/config';
+import { saveArticles } from '../../src/util/saveArticles';
 import { ZimArticle } from '@openzim/libzim';
 
 const redis = new Redis({ redis: process.env.REDIS }, config);

--- a/test/unit/urlRewriting.test.ts
+++ b/test/unit/urlRewriting.test.ts
@@ -7,11 +7,11 @@ require('dotenv').config();
 import test from 'blue-tape';
 import domino from 'domino';
 
-import { rewriteUrl } from 'src/util/rewriteUrls';
+import { rewriteUrl } from '../../src/util/rewriteUrls';
 import { makeLink, setupScrapeClasses } from 'test/util';
-import { populateArticleDetail, articleDetailXId, populateRedirects, redirectsXId } from 'src/stores';
-import Redis from 'src/redis';
-import { config } from 'src/config';
+import { populateArticleDetail, articleDetailXId, populateRedirects, redirectsXId } from '../../src/stores';
+import Redis from '../../src/redis';
+import { config } from '../../src/config';
 
 const redis = new Redis({ redis: process.env.REDIS }, config);
 populateArticleDetail(redis.redisClient);

--- a/test/util.ts
+++ b/test/util.ts
@@ -1,7 +1,7 @@
 import domino from 'domino';
-import MediaWiki from 'src/MediaWiki';
-import Downloader from 'src/Downloader';
-import { Dump } from 'src/Dump';
+import MediaWiki from '../src/MediaWiki';
+import Downloader from '../src/Downloader';
+import { Dump } from '../src/Dump';
 
 export function leftPad(_num: number, length: number) {
     const num = `${_num}`;

--- a/tsconfig.json
+++ b/tsconfig.json
@@ -25,14 +25,6 @@
       "es2017",
       "dom"
     ],
-    "paths": {
-      "src/*": [
-        "./src/*"
-      ],
-      "test/*": [
-        "./test/*"
-      ]
-    },
     "plugins": [
       {
         "name": "typescript-tslint-plugin",

